.dockerignore

```diff
@@ -61,6 +61,7 @@
 **/*.obj
 **/*.pch
 **/*.pdb
+!**/_.pdb
```

Comment:
It worth a comment with explanation why one kind of pdb is ignored while other allowed.